cmd/exposure/main.go

```diff
@@ -70,6 +70,9 @@ func realMain(ctx context.Context) error {
 	mux.Handle("/v1/publish", handler.Handle())
 	mux.Handle("/v1/publish/", http.NotFoundHandler())
 
+	// Handle metrics retrieval API
+	mux.Handle("/v1/metrics", handler.HandleMetrics())
```

Comment:
Should this be /stats instead? I think this is the only time it's called "metrics" and that word tends to mean something else in the context of this system.

Reply:
renamed - and all associated.